src/foundry/client-esm/canvas/tokens/ring.d.mts

```diff
@@ -150,43 +183,100 @@ declare class TokenRing {
   /**
    * To avoid breaking dnd5e.
    * @deprecated since v12
+   * @remarks No deprecation warning or end of deprecation period provided, method body completely empty
    */
   configureMesh(): void;
 
   /**
    * To avoid breaking dnd5e.
    * @deprecated since v12
+   * @remarks No deprecation warning or end of deprecation period provided, method body completely empty
    */
   configureNames(): void;
 }
 
 declare namespace TokenRing {
-  type SampleShaderClass = CONFIG["Token"]["ring"]["shaderClass"];
+  interface Any extends AnyTokenRing {}
+  type AnyConstructor = typeof AnyTokenRing;
+
+  /** @remarks Overrides for default values */
+  interface FlashColorOptions extends CanvasAnimationOptions {
+    /** @defaultValue `1600` */
+    duration?: number;
+
+    /** @defaultValue `PIXI.UPDATE_PRIORITY.HIGH` */
+    priority?: PIXI.UPDATE_PRIORITY | null | undefined;
+
+    /** @defaultValue `TokenRing.createSpikeEasing(0.15)` */
+    easing?: CanvasAnimation.EasingFunction | null | undefined;
+  }
+
+  interface TextureData {
+    UVs: Float32Array;
+    center: { x: number; y: number };
+  }
+
+  /**
+   * @remarks {@link TokenRing.configureSize | `configureSize`} effectively calls `Object.assign(this, RingData)`
```

Comment:
Include the class name in the code comment, as in ``{@link TokenRing.configureSize | `TokenRing.configureSize`}``